R/plotAbundance.R

```diff
@@ -129,6 +133,7 @@ setGeneric("plotAbundance", signature = c("x"),
 #' @rdname plotAbundance
 #' @importFrom scater plotExpression
 #' @importFrom ggplot2 facet_wrap
+#' @importFrom gridExtra grid.arrange
```

Comment:
We recommend the **patchwork** pkg for multi-panel plots. Could you change this?